content/ui-patterns/navigation.mdx

```diff
@@ -0,0 +1,392 @@
+---
+title: Navigation
+description: Navigation patterns help users understand where they are and where they can go next.
+---
+import {Box} from '@primer/react'
+
+This documentation provides an overview of navigation design patterns within Primer. For more granular guidance about the components and insights into their implementation, reference their respective guidelines pages.
+
+## Best practices
+
+Prioritize clarity and simplicity to facilitate seamless user interaction.
+
+### Clear user orientation
+
+Users should always have a clear understanding of their location within the application. Intuitive navigational cues minimize ambiguity.
+
+### Streamlined choices
+
+Minimize the number of navigational elements while ensuring that the most essential items are available. This balance can help prevent users from feeling overwhelmed with too many options, but also ensure that they can easily find what they need.
+
+### Contextual transitions
+
+Strive to maintain a balance between contextual clarity and minimal interruption. While transitions such as page refreshes or showing a dialog are useful for indicating context switches, excessive transitions can lead to unnecessary cognitive load.
+
+## Contextual shifts and transitions
+
+### Entirely new page
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{alignItems: 'flex-start', gap: 3}}>
+
+<Box flex={1}>
+
+This is the default behavior of clicking a link or submitting a form. The URL changes and the requested page loads.
+
+<!-- TODO: ask an engineer to confirm this is technically accurate -->
+In GitHub's single-page app architecture, this behavior is emulated without a traditional page reload.
+
+If you're navigating from a parent page to a child page, the child page should have a clear way to navigate back to the parent page. This is often done with [breadcrumbs](/components/breadcrumbs) or a "back" button.
+
+</Box>
+
+<CustomVideoPlayer
+  width="456"
+  loop
+  src="https://github.com/primer/design/assets/2313998/6d8008b5-bc5a-4f21-ba14-ac57550b2de3"
+/>
+
+</Box>
+
+For example, when adding a new Codespace secret, the user is taken to a new page to fill out the secret details. There is a breadcrumb at the top of the page letting them go back to the Codespace settings page.
+
+Be mindful of data loss when a user is navigating away from a page where they have unsaved changes. See the [saving](/ui-patterns/saving) guidelines for more info on handling unsaved changes.
+
+
+### Same page, different state
+
+<Box display="flex" flexDirection={['column', 'column', 'column', 'column', 'row']} sx={{alignItems: 'flex-start', gap: 3}}>
+
+<Box flex={1}>
+
+When users activate a different state of their current page. When using this kind of navigation, strive to avoid full page reloads when feasible. This could help maintain a fast and smooth user experience.
+
```

Comment:
```suggestion
Example: user activated a different state of their current page. 

When using this kind of navigation, strive to avoid full page reloads when feasible. This could help maintain a fast and smooth user experience.

```

These sections feel a little bit awkward with the heading and first sentence kind of describing the heading 🤔 this is a nitpick. But I'll try and suggest what I think would make sense and feel free to ignore!